### ISSUE TITLE
[BUG] {Solved unexpected shut down but getting left and right images makes FPS drop}RPi4 makes error unexpectedly

### ISSUE THREAD

**Issue by HanDaSeul** [luxonis/depthai-ros#301]
I'm using Raspberry Pi4 8GB, using OAK-D pro, ubuntu 20.04 with ros2 foxy with rmw_fastrtps and ust let it publish only three: left and right stereo compressed image, imu. Through WiFi, I can subscribe both image approximately 30FPS, imu to 100~200Hz. But, after the several minutes,  all the published topics were gone and it seemed RPi4 loose all the connection like shuts down itself so that I couldn't even reconnect remote. I made sure there's no electrical problem, put all the things up to 3A, respectively(Pi to 3A, Oak-d pro to 3A using Y-shapped).  

At the beginning:
![Screenshot from 2023-04-30 13-01-40](https://user-images.githubusercontent.com/32993131/235335626-a62b0b58-7aa5-4789-b586-f8f387b99eb0.png)

After the several minutes:
![Screenshot from 2023-04-30 13-34-00](https://user-images.githubusercontent.com/32993131/235335658-0bd2c2ac-e327-40b9-8061-53be96c9a1b0.png)

**Comment by HanDaSeul** [comment]
Update. I think I solved unexpected shut down. I moved all the usb devices to the usb hub, so the only Oak-d pro and usb hub are connected, and that usb hub need to have additional power. It can keep more than 40 minutes. 
So, I made RPi consumes up to 3A, Oak-d consumes up to 3A, usb hub consumes up to 2A. That means up to a total of 8A.
Intel t265 had same issue when I subscribe only pose data neither images or imu but I solved like that. 

At the beginning with usb hub, started at 13:47 :
![Screenshot from 2023-04-30 13-47-31](https://user-images.githubusercontent.com/32993131/235337456-f75d6716-c744-4f80-b2b5-31b8170e622c.png)

After the 40 minutes, at 14:29 :
![Screenshot from 2023-04-30 14-29-24](https://user-images.githubusercontent.com/32993131/235337473-d3168b82-2286-42e1-a1b5-f9da6fa5df54.png)

Here's the one weird thing. Subscribing one single image can stream 30FPS or 40FPS when I modified yaml file. 
But whenever I get other side image, FPS drops slightly about 1 to 6. When I set 40FPS, FPS drops 10 to 20.  Above screenshot is the case of 40FPS.
